settings/common.py

```diff
@@ -48,8 +48,12 @@
     'wye.workshops',
 )
 
-THIRD_PARTY_APPS = ('rest_framework',
-                    )
+THIRD_PARTY_APPS = (
+    'rest_framework',
+    'allauth',
+    'allauth.account',
```

Comment:
we need to add google, github and twitter logins option. Let me know if you want to open Issue and assign to you ?